### DISCUSSION TITLE
firefox: /opt is accessible

### DISCUSSION THREAD

**Discussion by brewhacker** [netblue30/firejail#4905] [Q&A] [answered]
Hello, 

So, maybe I don't quite fully understand how this is supposed to work.  I want to ensure I do understand or ensure I don't have a false sense of security if somethings wrong.  

So, I enabled Firejail on my Firefox thinking it would be completely shutoff from the rest of the system (sandboxed).   Below is the full command I'm using. 

When I open "file:///" in the browser window I'm able to traverse the file file system, and I am sure it's the live (not sandboxed) file system because I can go into my /opt dir and see apps I've installed there. I cannot however see my home directory, I do get a sandboxed version of that (which is a step in the right direction). 

Am I doing something wrong?  Or am I misunderstanding this?  

firejail --name=firefox --private-home=.mozilla --noexec=/tmp --nogroups --nonewprivs --apparmor firefox -no-remote -private-window -P YouTube

![Untitled](https://user-images.githubusercontent.com/51850593/152689614-a8f82c5f-ce65-46c6-a6f4-89d4158f2330.jpg)

**Comment by rusty-snake** [comment, marked as answer]
> and I am sure it's the live (not sandboxed) file system because I can go into my /opt dir and see apps I've installed there.

/opt isn't restricted by firefox.profile by default.

**Comment by brewhacker** [comment]
@rusty-snake  thanks.  So, I was using /opt as an example, but good to know.  That said, I can access just about every other directory too (lib, etc, bin, proc, etc).  Isn't one of the advantages of Firejail to protect system files from malicious activity (especially in a web browser)?

Appreciate the feedback.

**Reply by rusty-snake** [reply]
You need to system to run firefox ;)

And there are restriction of the system, you can not write to it for example.

**Reply by brewhacker** [reply]
OK, that's true I guess.  So, I guess it's working as intended.  Thanks.
